src/styles/blocks/calendar.scss

```diff
@@ -1,17 +1,38 @@
 .calendar {
-  width: 706px;
+  width: $size*7 + $padding*2 + $gap*6;
+  padding: 10px;
   display: flex;
   flex-flow: row wrap;
   gap: 1px;
 
   &__day {
     @extend %cell-style;
     transition: 0.5s;
+    cursor: pointer;
 
     &:hover {
       transform: translateY(-10px);
-      cursor: pointer;
       background-color: pink;
     }
   }
 }
+
+@each $day, $gap in $days {
+  .calendar__day--#{$day} >
+  .calendar__day:first-child {
```

Comment:
You could nest it to `&__day`, this way you should'n write `.calendar__day` only `&`.

More details in  [this video](https://mate.academy/learn/css-advanced/sass#/video/925) 2:44 timing